### DISCUSSION TITLE
[Feature] [app] Support additional parameters to HTTP header for Cloudflare Access authentication

### DISCUSSION THREAD

**Discussion by thariq-shanavas** [immich-app/immich#6432] [Feature Request]
### The feature

Hello!

I secure my immich instance using Cloudflare access. Cloudflare Access inserts a login page before the web app or API can be accessed over https.

There is no easy way to make this setup work with the app as of v1.92. I think the most straightforward way to add support for cloudflare access to the Immich app would be by generating a service token and adding it to the http header. The app would also need to support JWT if I'm understanding this correctly.

https://developers.cloudflare.com/cloudflare-one/identity/service-tokens/

![image](https://github.com/immich-app/immich/assets/19273195/dc0a345a-a920-4fe3-9bd9-cf6ff0474bea)

I would greatly appreciate if this feature could be added to the app. Thank you!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1604
